### PR TITLE
Bug: Typescript tsc could not be located when building node in Node-Dev

### PR DESCRIPTION
**Pull request by Rupenieks**
n8n now checks if user is using windows or otherwise before assigning path to tsc. 

Tested on:
- Linux subsystem (Ubuntu)
- Windows 10 

Thanks to @ivov for reporting and providing explanation (and essentially fix).

### CODE REVIEW

File: packages/node-dev/src/Build.ts [hunk 1]
```diff
@@ -63,8 +63,15 @@ export async function createCustomTsconfig () {
...
-	const tscPath = join(__dirname, '../../node_modules/.bin/tsc');
+	let typescriptPath;
+
+	// Check for OS to designate correct tsc path
+	if (process.platform === 'win32') {
+		typescriptPath = '../../node_modules/TypeScript/lib/tsc';
+	} else {
+		typescriptPath = '../../node_modules/.bin/tsc';
+	}
+    const tscPath = join(__dirname, typescriptPath);
```
Comment (janober) [inline packages/node-dev/src/Build.ts:74]: Looks like there are spaces again.

Comment (Rupenieks) [reply packages/node-dev/src/Build.ts]: @janober Fixed!

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.81.0

**Pull request merged**